types/upyun/index.d.ts

```diff
@@ -0,0 +1,331 @@
+// Type definitions for upyun 3.4
+// Project: https://github.com/upyun/node-sdk/
+// Definitions by: Zero <https://github.com/groupguanfang>, Leigh <https://github.com/upyun>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+/// <reference types="node" />
+
+import buffer = require('buffer');
```

Comment:
is this `node:buffer` (and can you use scoped imports?, there is a `buffer` package)